graphql/resolvers.go

```diff
@@ -2410,6 +2410,21 @@ func (r *ticketFieldsResolver) ResolutionName(ctx context.Context, obj *thirdpar
 
 func (r *Resolver) TicketFields() TicketFieldsResolver { return &ticketFieldsResolver{r} }
 
+func (r *taskResolver) Annotation(ctx context.Context, obj *restModel.APITask) (*restModel.APITaskAnnotation, error) {
+	annotation, err := annotations.FindOneByTaskIdAndExecution(*obj.Id, obj.Execution)
+	if err != nil {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error finding annotation: %s", err.Error()))
+	}
+	if annotation == nil {
+		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("Unable to find annotation %s", *obj.Id))
```

Comment:
I'm not convinced this should be an error. if no annotation exists for a taskId and execution, the page that uses this resolver should just display no annotation, not display an error. Does it make sense to case on that where this query is used? Or to just handle that here and return an empty annotation?

Reply:
the return type for this query is nullable so it is okay to return `nil, nil` instead of an empty annotation.

Reply:
good point, switched it to return null